test/format/config.sh

```diff
@@ -20,12 +20,13 @@ typedef struct {
 
 #define C_BOOL 0x001u        /* Boolean (true if roll of 1-to-100 is <= CONFIG->min) */
 #define C_IGNORE 0x002u      /* Not a simple randomization, configured specially */
-#define C_STRING 0x004u      /* String (rather than integral) */
-#define C_TABLE 0x008u       /* Value is per table, not global */
-#define C_TYPE_FIX 0x010u    /* Value is only relevant to FLCS */
-#define C_TYPE_LSM 0x020u    /* Value is only relevant to LSM */
-#define C_TYPE_ROW 0x040u    /* Value is only relevant to RS */
-#define C_TYPE_VAR 0x080u    /* Value is only relevant to VLCS */
+#define C_NEUTRAL 0x004u     /* Does not affect data written to disk */
```

Comment:
I am a little worried about future developers either not knowing about this or forgetting to include it. What is the result/risk of someone not setting this when it should be set? Is there a clean way to force the developer to specify something? I.e. create two defines `C_DATA_TOUCH` and `C_DATA_AVOID` (or something) and require that one or the other be set always. That forces the developer to think about it.

Reply:
As mentioned below, I removed the premise of having a "predictable" configuration be generated.  So `C_NEUTRAL` is gone!